wgengine/router/router_windows.go

```diff
@@ -29,6 +30,7 @@ type winRouter struct {
 	routeChangeCallback *winipcfg.RouteChangeCallback
 	dns                 *dns.Manager
 	firewall            *firewallTweaker
+	wgFirewallEnabled   bool
```

Comment:
this is the only mutable field, right? separate out by a blank link and document the concurrency rules? owned by Set? which is not called concurrently due to .... ?

Reply:
Separated and documented that wgengine serializes calls to router.